components/TextFields/src/ContainedInputView/private/MDCTextControlStyleFilled.m

```diff
@@ -150,6 +153,11 @@ - (void)applyStyleToView:(UIView *)view
                    state:(MDCTextControlState)state
           containerFrame:(CGRect)containerFrame
        animationDuration:(NSTimeInterval)animationDuration {
+  BOOL aBoundsChangeHasTakenPlace = NO;
+  if (!CGRectEqualToRect(self.mostRecentBounds, view.bounds)) {
+    aBoundsChangeHasTakenPlace = YES;
```

Comment:
Should we be setting the mostRecentBounds property here? Otherwise my under standing of this is that it will always be zero.

Reply:
Yes haha whoops. Thanks.